actions/structs.go

```diff
@@ -2,48 +2,125 @@ package actions
 
 import (
 	"github.com/AnomalyFi/hypersdk/codec"
+	"github.com/AnomalyFi/hypersdk/consts"
 	"github.com/AnomalyFi/hypersdk/utils"
 	"github.com/ava-labs/avalanchego/ids"
 )
 
-// TODO: do we need to rename it to `RollupInfo`
-type AnchorInfo struct {
-	FeeRecipient codec.Address `json:"feeRecipient"`
-	Namespace    []byte        `json:"namespace"`
+type RollupInfo struct {
+	FeeRecipient        codec.Address `json:"feeRecipient"`
+	Namespace           []byte        `json:"namespace"`
+	AuthoritySEQAddress codec.Address `json:"authoritySEQAddress"`
```

Comment:
What does AuthoritySEQAddress do here?

Reply:
has access to update or delete rollup info.